skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -74,6 +74,7 @@
 <?= $dynamicYield ?>
 <?= $ivw2 ?>
 <?= $rubiconRtp ?>
+<gcse:search></gcse:search>
```

Comment:
If this is for A/B test it would be actually better to have everything in one place so ones test is over it could be easily removed from our code base. 

If this can not be added to DOM via JS please consider adding comment here with TODO and link to jira ticket to remove `<gcse:search></gcse:search>` ones the test is over